content/03-reference/01-tools-and-interfaces/07-preview-features/001-sql-server/010-sql-server-connection-string.mdx

```diff
@@ -0,0 +1,64 @@
+
+</TopBlock>
+
+
+## Arguments
+
+| Argument name                                                 | Required          | Default  | Comments                                                                                                                                                                                                                                                                                                               |
+| :------------------------------------------------------------ | :---------------- | :------- | :----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `database`<br />`initial catalog`                             | No                | `master` | The database to connect to.                                                                                                                                                                                                                                                                                            |
+| `username`<br />`user`<br />`uid`<br />`userid`               | No - see Comments |          | SQL Server login (such as `sa`) *or* a valid Windows (Active Directory) username if `Integrated Security` is set to `true` (Windows only).                                                                                                                                                                             |
+| `password`<br />`pwd`                                         | No - see Comments |          | Password for SQL Server login *or* Windows (Active Directory) username if `Integrated Security` is set to `true` (Windows only).                                                                                                                                                                                       |
+| `encrypt`                                                     | No                |          | Configures whether to use TLS, possible values: `true`, `false`, `DANGER_PLAINTEXT`                                                                                                                                                                                                                                    |
+| `Integrated Security`                                         | No                |          | Enables Windows integrated authentication, possible values: `true`, `false`, `yes`, `no`. If set to `true` or `yes` and `username` and `password` are present, login is performed through Windows Active Directory. If login details are not given, the current logged in Windows user is used to login to the server. |
```

Comment:
```suggestion
| `Integrated Security`                                         | No                |          | Enables Windows integrated authentication, possible values: `true`, `false`, `yes`, `no`. If set to `true` or `yes` and `username` and `password` are present, login is performed through Windows Active Directory. If login details are not given via separate arguments, the current logged in Windows user is used to login to the server. |
```

Reply:
Also the key has no whitespace. `IntegratedSecurity`